.travis.yml

```diff
@@ -12,4 +12,4 @@ before_script: composer install --ignore-platform-reqs
 
 script:
 - vendor/bin/phpunit --configuration phpunit.xml < tests/input.txt 
-- vendor/bin/psalm --show-info=true
+- vendor/bin/phpcs -p
```

Comment:
Should we change this?

Reply:
Yes as we are using phpcs in Appwrite,  and everywhere.

Reply:
OK 👍